src/System.Security.Permissions/ref/project.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "System.Collections.NonGeneric": "4.0.0",
+    "System.Reflection": "4.1.0",
+    "System.Runtime.Extensions": "4.1.0",
+    "System.Runtime": "4.2.0-beta-devapi-24415-01",
```

Comment:
I believe all the "244xx" numbers in your project.json's need to match the number of the day, which today is     "System.Runtime": "4.2.0-beta-devapi-24418-01"